### ISSUE TITLE
How to prepare latents when training process

### ISSUE THREAD

**Issue by lgs00** [Zheng-Chong/CatVTON#43]
latens are random initialization at test time, but when training, how to prepare latens so that it has the same dimension as mask_latens? Is it the same as mask below?
`latent = torch.cat([latent, torch.zeros_like(latent)], dim=concat_dim)`
